types/module/module.go

```diff
@@ -195,9 +195,6 @@ func (GenesisOnlyAppModule) RegisterInvariants(_ sdk.InvariantRegistry) {}
 // QuerierRoute returns an empty module querier route
 func (GenesisOnlyAppModule) QuerierRoute() string { return "" }
 
-// LegacyQuerierHandler returns an empty module querier
```

Comment:
Can we remove this line then too: https://github.com/cosmos/cosmos-sdk/blob/77a96d5fe14076cdbd4eeb7f1b29cb5687ed495f/docs/building-modules/module-manager.md#L73